pages/pipelines/environment_variables.md.erb

```diff
@@ -406,6 +406,12 @@ The repository of your pipeline. This variable can be set by exporting the envir
 
 Example: `"git@github.com:acme-inc/my-project.git"`
 
+<h3 class="h3-caps">BUILDKITE_REPO_MIRROR</h3>
+
+Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled. The path to the shared git mirror. The value cannot be modified.
```

Comment:
“The value cannot be modified” feels like it lacks context.
Maybe “… during a build” or “… after the agent has started” or something.
But I see this is an existing pattern, so maybe not important/relevant for this particular PR.

Reply:
@pda yeah, that's mostly why I didn't mess with it. But I'd like to be more specific about what environment variables are writable when. 

What is the best way to get that info on a systematic basis?